doc/source/install.rst

```diff
@@ -26,4 +26,5 @@ size data.
    install/fedora
    install/solaris
    install/docker
+   install/cmake
```

Comment:
How about removing this and refer `install/cmake` from `install/others.rst` instead?

Reply:
I have removed `install/cmake` from this and make `cmake.rst` orphan.

> refering install/cmake from install/others.rst instead?

I will Immediately do this after this patch merged.
When separating `install/others.rst` into `autotools.rst`, I will also do that.